Makefile

```diff
@@ -63,45 +83,52 @@ test:
 	./hack/coverage.sh $(GINKGO) $(TARGETCOVERAGE)
 
 gen-k8s: $(apis_sources)
-	./hack/gen-k8s.sh generate k8s
+	./hack/gen-k8s.sh
 
 gen-k8s-check: $(apis_sources)
 	./hack/verify-codegen.sh
 
 gen-crds: $(apis_sources)
 	./hack/gen-crds.sh
 
-container-build: container-build-operator container-build-registry
+container-build: container-build-operator container-build-bundle container-build-index
 
 container-build-operator: csv-generator
 	docker build -f build/Dockerfile -t $(IMAGE_REGISTRY)/$(OPERATOR_IMAGE):$(IMAGE_TAG) .
 
-container-build-registry:
-	docker build -f build/Dockerfile.registry -t $(IMAGE_REGISTRY)/$(REGISTRY_IMAGE):$(IMAGE_TAG) .
+container-build-bundle:
```

Comment:
Not a prereq for this PR, but could we add a comment with links to what the bundle and index parts are for?
Helpful for those that come after us :)

Reply:
fair point, the readme etc. probably needs an update anyway, forgot about it